File: library/src/main/java/com/alamkanak/weekview/EventClickListener.java
package com.alamkanak.weekview;

import android.graphics.RectF;

public interface EventClickListener {
    /**
     * Triggered when clicked on one existing event
     *
     * @param event:     event clicked.
     * @param eventRect: view containing the clicked event.
     */
    void onEventClick(WeekViewEvent event, RectF eventRect);
}

File: library/src/main/java/com/alamkanak/weekview/EventClickListener.kt
package com.alamkanak.weekview

import android.graphics.RectF

interface EventClickListener {
    /**
     * Triggered when clicked on one existing event
     *
     * @param event:     event clicked.
     * @param eventRect: view containing the clicked event.
     */
    fun onEventClick(event: WeekViewEvent, eventRect: RectF)
}
